manifests/l/Lenovo/ThinInstaller/1.3.0038/Lenovo.ThinInstaller.installer.yaml

```diff
@@ -3,6 +3,14 @@
 
 PackageIdentifier: Lenovo.ThinInstaller
 PackageVersion: 1.3.0038
+MinimumOSVersion: 6.1
+Scope: machine
+InstallModes:
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT

```

Comment:
The installerswitches are not needed as its already specified as Inno

Reply:
Thanks.  Lines have been removed.